client/i18n/locales/english/translations.json

```diff
@@ -390,7 +390,7 @@
     "running-tests": "// running tests",
     "tests-completed": "// tests completed",
     "console-output": "// console output",
-    "syntax-error": "Your code raised an error before any tests could run. Please fix it and try again.",
+    "syntax-error": "Your code has a syntax error, see the console for details.",
```

Comment:
syntax error is too specific for python, there are many different errors possible (TypeError, ValueError etc). Also it's terminal, not console. Maybe it needs two different strings?

Reply:
Got it chief.